packages/web/src/components/other/page_not_found.css

```diff
@@ -94,8 +94,8 @@
     .hubzie-UFO-image {
         width: 100%;
         justify-content: center;
-        margin-top: 5%;
-        margin-bottom: 5%;
+        margin-top: 10%;
+        margin-bottom: 1%;
```

Comment:
Everything LGTM, just please add a little more margin bottom for the hubzie-UFO-image because in all devices, the 404 text seems too close to the image. After being done please resolve conversations and re-request review from me.